docs/examples/asyncagi/example.php

```diff
@@ -48,8 +47,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 // Mandatory stuff to bootstrap.
 ////////////////////////////////////////////////////////////////////////////////
-require_once 'PAMI/Autoloader/Autoloader.php'; // Include PAMI autoloader.
-\PAMI\Autoloader\Autoloader::register(); // Call autoloader register for PAMI autoloader.
+require_once 'autoload.php'; // Include Composer autoloader.
```

Comment:
I would just use `__DIR__.'/../../../vendor/autoload.php'`. Makes it immediately clear where to look for the file.